src/datadoc/backend/unit_types.py

```diff
@@ -73,7 +153,7 @@ def _get_classification_dataframe_if_loaded(self) -> bool:
         return False
 
     @property
-    def classifications(self) -> list[str]:
+    def classifications(self) -> list[UnitType]:
         """Getter for primary subjects."""
         self._get_classification_dataframe_if_loaded()
```

Comment:
```suggestion
        if not self._classifications:
            self._get_classification_dataframe_if_loaded()
```

We only want to process this once, not every time the classifications are accessed.